R/db-postgres.r

```diff
@@ -21,6 +21,16 @@ sql_translate_env.PostgreSQLConnection <- function(con) {
         digits <- as.integer(digits)
         sql_expr(round((!!x) %::% numeric, !!digits))
       },
+      grepl  = function(pattern, x, ignore.case = FALSE, perl = FALSE, fixed = FALSE, useBytes = FALSE) {
+        if (any(c(perl, fixed, useBytes))) {
+          stop("perl, fixed and useBytes parameters are unsupported")
+        }
+        if (ignore.case) {
```

Comment:
Would you mind including a link to the corresponding docs in a comment?

Reply:
Do you mean link to https://www.postgresql.org/docs/current/static/functions-matching.html#FUNCTIONS-POSIX-TABLE?

Reply:
Yes, perfect. It's helpful for me when I re-read the code in the future because I forget all the details of the different SQL dialects